src/showcase-merge/app/angular/icon-overview/cdn-icon-list/cdn-icon-list.component.html

```diff
@@ -18,5 +18,9 @@
 <div [class.sbb-icon-fit]="filterForm.get('fitIcons').value">
   <sbb-cdn-icon *ngFor="let cdnIcon of filteredIcons | async" [cdnIcon]="cdnIcon"></sbb-cdn-icon>
 </div>
-<sbb-paginator [pageSize]="pageSize"></sbb-paginator>
+<sbb-paginator
+  [pageSize]="pageSize"
+  aria-label="Select page of icons"
```

Comment:
I'm not sure about this. Does the following sound "smoother"?
```suggestion
  aria-label="Select icon page"
```